.travis.yml

```diff
@@ -1,8 +1,15 @@
 language: java
 install: true
 dist: trusty
+
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install jq
+  - wget -O ~/codacy-coverage-reporter-assembly-latest.jar $(curl https://api.github.com/repos/codacy/codacy-coverage-reporter/releases/latest | jq -r .assets[0].browser_download_url)
+
 after_success:
-- "./gradlew coveralls"
+- gradle jacocoRootReport 
```

Comment:
./gradlew

Reply:
I have already set the github and the codacy tokens for the travis of opt4j. As I have write access to this repo, this should be enough, but we can also replace it by your gitHub token. One thing I cannot do is allowing codacy to have a hook for commits and pushes to the repo, as I am not the repository admin. Could you please do that? You just have to go to codacy, login with gitHub, go opt4j/settings/integrations and set the checkbox for Post-Commit Hook